docset/windows/hyper-v/set-vmnetworkadaptervlan.md

```diff
@@ -103,11 +103,18 @@ PS C:\> Get-VM Redmond | Set-VMNetworkAdapterVlan -Untagged
 
 Gets virtual machine Redmond and sets the virtual network adapters in the virtual machine to the untagged mode.
 
+### Example 5
+```
+PS C:\> Set-VMNetworkAdapterVlan -ManagementOS -Access -VlanID 20
+```
+
+Sets the virtual switch in the management OS to the Access mode.
+Traffic sent by this virtual switch is tagged with VLAN ID 20.
+
 ## PARAMETERS
 
 ### -Access
-Specifies **Access** mode for the virtual machine network adapter.
-This parameter must be specified in conjunction with parameter **VlanId**.
+Specifies **Access** mode for the virtual machine network adapter. This parameter configures an untagged virtual port with the VLANId (port based VLAN), for this reason must be specified in conjunction with parameter **VlanId**.
```

Comment:
```suggestion
Specifies **Access** mode for the virtual machine network adapter. This parameter configures an untagged virtual port with the VLANId (port based VLAN), so it must be specified together with **VlanId**.
```

Reply:
I am not sure about this... Maybe:

"..., for this reason must be specified together with the **VlanId** parameter."

What do you think @get-itips @tseward @nenonix?

Reply:
Both are ok I think